tslint.json

```diff
@@ -1,7 +1,6 @@
  
 {
   "rules": {
-    "arrow-parens": true,
```

Comment:
why did you need to remove this ?

Reply:
I think it was conflicting with another one of the style guides (probably gts)